homeassistant/components/system_bridge/sensor.py

```diff
@@ -283,15 +308,16 @@ async def async_setup_entry(
             )
 
     displays = []
-    for display in coordinator.data.display.displays:
-        displays.append(
-            {
-                "key": display,
-                "name": getattr(coordinator.data.display, f"{display}_name").replace(
-                    "Display ", ""
-                ),
-            },
-        )
+    if coordinator.data.display.displays is not None:
+        for display in coordinator.data.display.displays:
+            displays.append(
+                {
+                    "key": display,
+                    "name": getattr(
+                        coordinator.data.display, f"{display}_name"
+                    ).replace("Display ", ""),
```

Comment:
I assume this is to remove a `"Display "` prefix – since HA is Python 3.9+, you could [`removeprefix`](https://docs.python.org/3.10/library/stdtypes.html#str.removeprefix).

```suggestion
                    ).removeprefix("Display "),
```